From 099fbe881236fd0660edce720048eed64c36bc87 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Fri, 26 Oct 2018 18:35:54 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399pro-npu set armclk to 1200MHz

Set cpu clock to 1.2GHz and  vdd_cpu fixed to 0.85v by hardware.

Change-Id: Ib3d537a4ec4e419b36140d556771b3ba041dc24e
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399pro-npu-evb-v10.dts | 4 ++--
 arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi        | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro-npu-evb-v10.dts b/arch/arm64/boot/dts/rockchip/rk3399pro-npu-evb-v10.dts
index 5823aa1e9287..37d52fb8209d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro-npu-evb-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro-npu-evb-v10.dts
@@ -47,8 +47,8 @@
 		regulator-name = "vdd_cpu";
 		regulator-always-on;
 		regulator-boot-on;
-		regulator-min-microvolt = <1000000>;
-		regulator-max-microvolt = <1000000>;
+		regulator-min-microvolt = <850000>;
+		regulator-max-microvolt = <850000>;
 	};
 
 	vcc5v0_sys: vcc5v0-sys {
diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi b/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi
index 415c54474cfe..05d0d4fc2982 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi
@@ -249,7 +249,7 @@
 			<&cru LSCLK_BUS_PRE>;
 		assigned-clock-rates =
 			<1188000000>, <1000000000>,
-			<100000000>, <816000000>,
+			<100000000>, <1200000000>,
 			<200000000>, <100000000>,
 			<300000000>, <200000000>,
 			<100000000>;
-- 
2.35.3

